platforminputcontext/main.h

```diff
@@ -29,7 +29,7 @@ class QFcitxPlatformInputContextPlugin : public QPlatformInputContextPlugin
 {
     Q_OBJECT
 public:
-    Q_PLUGIN_METADATA(IID QPlatformInputContextFactoryInterface_iid FILE "fcitx.json")
+    Q_PLUGIN_METADATA(IID "QPlatformInputContextFactoryInterface_iid" FILE "fcitx.json")
```

Comment:
this is not right. This is actually a macro in ./QtGui/5.7.1/QtGui/qpa/qplatforminputcontextplugin_p.h